projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml

```diff
@@ -265,6 +265,10 @@ spec:
               value: "{{ .Values.dataJob.deployment.initContainer.resources.limits.memory }}"
             - name: DATAJOBS_DEPLOYMENT_INITCONTAINER_RESOURCES_LIMITS_CPU
               value: "{{ .Values.dataJob.deployment.initContainer.resources.limits.cpu }}"
+            {{- if .Values.dataJob.deployment.supportedPythonVersions }}
+            - name: DATAJOBS_DEPLOYMENT_SUPPORTED_PYTHON_VERSIONS
+              value: "{{ .Values.dataJob.deployment.supportedPythonVersions }}"
```

Comment:
I think you need toJson . https://helm.sh/docs/chart_template_guide/function_list/#tojson-musttojson 

How was this tested ?

I don't see where Values.dataJob.deployment.supportedPythonVersions is defined (values.yaml should be changed)

Reply:
This configuration should be exposed further on.

Reply:
Yes, ok that make sense. Let's add a task (issue) for that.

Reply:
Removed this for the time being and created an issue for it - https://github.com/vmware/versatile-data-kit/issues/1776